lib/normalizr/concern.rb

```diff
@@ -20,8 +20,8 @@ def normalize(*args, &block)
                 value = Normalizr.normalize(value, *options.before)
                 value = Normalizr.normalize(value, *options.after) if options.after.any?
               end
-
-              super(value)
+              args.insert(0, value)
+              super(*args)
```

Comment:
```suggestion
              super(value, *args)
```

I think we can pass the `value` variable as the 1st parameter to `super` without mutating the `args`.

Reply:
done